internal/dataplane/parser/translators/httproute_atc.go

```diff
@@ -227,3 +229,347 @@ func matchersFromParentHTTPRoute(hostnames []string, metaAnnotations map[string]
 	}
 	return ret
 }
+
+const (
+	InternalRuleIndexAnnotationKey  = "internal-rule-index"
+	InternalMatchIndexAnnotationKey = "internal-match-index"
+)
+
+// SplitHTTPRoute split HTTPRoutes into HTTPRoutes with at most one hostname, and at most one rule
+// with exactly one match. It will split one rule with multiple hostnames and multiple matches
+// to one hostname and one match per each HTTPRoute.
```

Comment:
```suggestion
// SplitHTTPRoute splits HTTPRoutes into HTTPRoutes with at most one hostname, and at most one rule
// with exactly one match. It will split one rule with multiple hostnames and multiple matches
// to one hostname and one match per each HTTPRoute.
```

Reply:
done.